doc/source/release/1.16.3-notes.rst

```diff
@@ -12,12 +12,37 @@ compared to 1.16.2.
 Authors
 =======
 * Name (commits)
+* Ralf Gommers (3)
+* Matt Haberland (1)
+* Nick ODell (1)
+* Ilhan Polat (1)
+* Tyler Reddy (14)
+
+    A total of 5 people contributed to this release.
+    People with a "+" by their names contributed a patch for the first time.
+    This list of names is automatically generated, and may not be fully complete.
 
 
 Issues closed for 1.16.3
 ------------------------
 
+* `#23336 <https://github.com/scipy/scipy/issues/23336>`__: BUG: linalg: wrong handling of args/kwargs in ``_apply_over_batch``
+* `#23469 <https://github.com/scipy/scipy/issues/23469>`__: BUG: huge speed and quality regression for stats.multivariate_normal.cdf
+* `#23642 <https://github.com/scipy/scipy/issues/23642>`__: CI: meson setup fails with "invalid int value: 'Sentinel.UNSET'"
```

Comment:
I see that xsf has been updated, which means that the fix (https://github.com/scipy/xsf/pull/62) for #23589 is also included:

```suggestion
* `#23589 <https://github.com/scipy/scipy/issues/23589>`__: BUG: ``lqmn(1,1,0)`` and  ``lqmn(1,1,0j)`` return incompatible derivatives
* `#23642 <https://github.com/scipy/scipy/issues/23642>`__: CI: meson setup fails with "invalid int value: 'Sentinel.UNSET'"
```

Reply:
Oh thanks--it didn't get slurped up by the automated release notes list scripts because it was lacking a milestone (I just added the `1.16.3` milestone over there).

That said, this does make me think about something else--I'm pretty sure the author list/name script won't pick you up as an author on the release since your patch was in a different repo.

My initial inclination is to be reasonably inclusive on authorship, maybe I can just add you manually in this case. Is there a feeling that `xsf` is sufficiently independent that folks working on it are happy to get author "credit" over there on its own releases? I don't have a good read on that--I suppose `xsf` has prospects for wide usage separate from SciPy anyway, so maybe?

Reply:
I would say that the primary visibility of authorship is commit counts, and `xsf` has its own (https://github.com/scipy/xsf/graphs/contributors). So I wouldn't do work to treat `xsf` differently from all other libraries we vendor.

Reply:
I was mostly talking about the resolved issue I raised in the scipy repo, actually